x-pack/plugins/osquery/cypress/e2e/all/alerts.cy.ts

```diff
@@ -272,16 +269,52 @@ describe('Alert Event Details', () => {
     cy.visit('/app/osquery');
     closeModalIfVisible();
   });
-  // TODO think on how to get these actions triggered faster (because now they are not triggered during the test).
-  // it.skip('sees osquery results from last action', () => {
-  //   cy.visit('/app/security/alerts');
-  //   cy.getBySel('header-page-title').contains('Alerts').should('exist');
-  //   cy.getBySel('expand-event').first().click({ force: true });
-  //   cy.contains('Osquery Results').click();
-  //   cy.getBySel('osquery-results').should('exist');
-  //   cy.contains('select * from uptime');
-  //   cy.getBySel('osqueryResultsTable').within(() => {
-  //     checkResults();
-  //   });
-  // });
+
+  it('should substitute parameters in investigation guide', () => {
+    cy.visit('/app/security/alerts');
+    cy.getBySel('header-page-title').contains('Alerts').should('exist');
+    cy.getBySel('expand-event')
+      .first()
+      .within(() => {
+        cy.get(`[data-is-loading="true"]`).should('exist');
+      });
+    cy.getBySel('expand-event')
+      .first()
+      .within(() => {
+        cy.get(`[data-is-loading="true"]`).should('not.exist');
+      });
+    cy.getBySel('expand-event').first().click({ force: true });
+    cy.contains('Get processes').click();
+    cy.contains("SELECT * FROM os_version where name='Ubuntu';");
+  });
+
+  it('sees osquery results from last action', () => {
+    toggleRuleOffAndOn(RULE_NAME);
+    cy.wait(2000);
+    cy.visit('/app/security/alerts');
+    cy.getBySel('header-page-title').contains('Alerts').should('exist');
+    cy.getBySel('expand-event').first().click({ force: true });
+    cy.contains('Osquery Results').click();
+    cy.getBySel('osquery-results').should('exist');
+    cy.contains('select * from uptime');
+    cy.contains('select * from users;');
+    cy.contains("SELECT * FROM os_version where name='Ubuntu';");
+    cy.getBySel('osquery-results-comment').each(($comment) => {
+      cy.wrap($comment).within(() => {
+        // On initial load result table might not render due to displayed error
+        if ($comment.find('div .euiDataGridRow').length <= 0) {
```

Comment:
Interesting workaround, have you seen this one? ;) https://github.com/elastic/kibana/blob/00a7cf6cb88d51965f7c09c190f20bb5d34184ee/x-pack/plugins/osquery/cypress/tasks/live_query.ts#L34

Reply:
Yes, I have. Alas, the `checkResults` function expects these two tabs to be present (therefore if table is not present it will try to toggle between Status and Results). In our case we end up with packs result that has no Status and Results tabs and test was failing. 
![Screenshot 2023-02-02 at 14 38 57](https://user-images.githubusercontent.com/29123534/216340450-961035c2-3c87-48ec-a9ad-a1e337c5c3e0.png)

Another thing that we will need to look at is the packs table itself as it seems not to have any selectors implemented and effectively is not being tested with this test.